src/subcommand/wallet/inscribe.rs

```diff
@@ -14,14 +14,33 @@ use {
 
 #[derive(Debug, Parser)]
 pub(crate) struct Inscribe {
+  #[clap(long, help = "Inscribe on <ORDINAL>")]
   ordinal: Ordinal,
-  content: String,
+  #[clap(long, help = "Inscribe contents of <FILE>")]
+  file: PathBuf,
 }
 
 impl Inscribe {
   pub(crate) fn run(self, options: Options) -> Result {
     let client = options.bitcoin_rpc_client_mainnet_forbidden("ord wallet inscribe")?;
 
+    let inscription = match &self
+      .file
+      .extension()
+      .expect("file should always have an extension")
```

Comment:
Files don't always have extensions, so we should handle this error case. Something like:

```suggestion
      .ok_or_else(|| anyhow!("file must have extension"))?
      .to_str()
      .ok_or_else(|| anyhow!("unrecognized extension"))?
```